ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -973,7 +973,7 @@
     "Name": "Applicant's details",
     "Description": "Applicant's details",
     "DisplayOrder": 6,
-    "PreConditionState(s)": "Open;RETURNED",
```

Comment:
Doesn't the LA need access to the Applicant's details in open state to create a case?

Reply:
You're right, just confirming with BA